src/frontend/src/catalog/system_catalog/mod.rs

```diff
@@ -292,7 +292,7 @@ fn get_acl_items(
 }
 
 pub struct SystemCatalog {
-    // table id = index + 1
+    // table id = index + SYS_CATALOG_START_ID
```

Comment:
A bit weird to me, but acceptable

Reply:
In order to be compatible with sql backend.